lib/Pipeline.js

```diff
@@ -84,6 +85,9 @@ Pipeline.processJSONWithExtras  = function(gltfWithExtras, options) {
     if (options.removeNormals) {
         removeNormals(gltfWithExtras);
     }
+    updateVersion(gltfWithExtras, {
+        targetVersion: '1.0'
```

Comment:
For now, this is set to `1.0` intentionally since I don't think we want to be generating `1.1` from master by default just yet. Though now that I think about it, command flag for `1.1` may be a better solution.